### PR TITLE
CORE-33390 - adjust configuration options names

### PR DESCRIPTION
**Pull request by artur-ciocanu**
Adjust configuration options names

## Additional Details

Adjust configuration options names

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by artur-ciocanu** [comment]
@jfkhoury that's cool, let's wait.

File: fundamentals/configuring-the-sdk.md [hunk 1]
```diff
@@ -44,19 +44,19 @@ Your assigned Experience Cloud organization ID.
...
 **Required:** No\
 **Default Value:** `false`
 
 Indicates whether debugging messages should be displayed in the browser's JavaScript console. This setting is sticky and will persist until you change it. 
 
-### `suppressErrors`
+### `errorsEnabled`
  **Type:** Boolean\
  **Required:** No\
  **Default Value:** `false`
```
Comment (dancingcactus) [inline fundamentals/configuring-the-sdk.md]: Wouldn't the default value be true in this case?

File: fundamentals/configuring-the-sdk.md [hunk 2]
```diff
@@ -44,19 +44,19 @@ Your assigned Experience Cloud organization ID.
...
 
-Indicates whether errors should be suppressed. As described in [Executing Commands](executing-commands.md), _uncaught_ errors will be logged to the developer console, regardless of whether logging is enabled in Alloy. By setting `suppressErrors` to `true`, promises returned from Alloy will never be rejected, though errors will still be logged to the console if logging is enabled in Alloy. 
+Indicates whether errors should be suppressed. As described in [Executing Commands](executing-commands.md), _uncaught_ errors will be logged to the developer console, regardless of whether logging is enabled in Alloy. By setting `errorsEnabled` to `true`, promises returned from Alloy will never be rejected, though errors will still be logged to the console if logging is enabled in Alloy.
```
Comment (dancingcactus) [inline fundamentals/configuring-the-sdk.md]: setting `errorsEnabled` to `true` > setting `errorsEnabled` to `false`

File: fundamentals/configuring-the-sdk.md (hunk 1 above)
Comment (artur-ciocanu) [reply fundamentals/configuring-the-sdk.md]: Fixed.

File: fundamentals/configuring-the-sdk.md (hunk 2 above)
Comment (artur-ciocanu) [reply fundamentals/configuring-the-sdk.md]: Fixed.

**Pull request merged**